packages/material-ui-codemod/transforms/v0.15.0/import-path.js

```diff
@@ -0,0 +1,69 @@
+const pathConversion = {
+  'app-bar': 'AppBar',
+  'auto-complete': 'AutoComplete',
+  'avatar': 'Avatar',
+  'paper': 'Paper',
+  'tabs/tab': 'Tabs/Tab',
+  'tabs/tabs': 'Tabs/Tabs',
+  'text-field': 'TextField',
+  'checkbox': 'Checkbox',
+  'snackbar': 'Snackbar',
+  'lists/list-item': 'List/ListItem',
+  'toggle': 'Toggle',
+  'dialog': 'Dialog',
+  'radio-button': 'RadioButton',
+  'raised-button': 'RaisedButton',
+  'flat-button': 'FlatButton',
+  'circular-progress': 'CircularProgress',
+  'select-field': 'SelectField',
+  'date-picker/date-picker': 'DatePicker',
+  'menus/menu-item': 'MenuItem',
+  'menus/icon-menu': 'IconMenu',
+  'floating-action-button': 'FloatingActionButton',
+  'icon-button': 'IconButton',
+  'MuiThemeProvider': 'styles/MuiThemeProvider',
+};
+
+const pathBaseSource = 'material-ui/src/';
+const pathBasePackage = 'material-ui/lib/';
```

Comment:
Non, that variable is for the existing code :grin:. But I get what you mean. As we plan do remove the `lib` from the require, we should reflect it in the codemod.